### PR TITLE
Make not found page

### PR DESCRIPTION
**Pull request by avolchivschi**
@andreikojusko

### CODE REVIEW

File: intellst-front/public/index.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900">
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@mdi/font@latest/css/materialdesignicons.min.css">
   </head>
+  <body>
```
Comment (andreikojusko) [inline intellst-front/public/index.html]: why do you want another body? the one you've got is already great! ;)

File: intellst-front/src/store/constants.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+const BaseUrl = "http://localhost/";
```
Comment (andreikojusko) [inline intellst-front/src/store/constants.js]: This shouldn't be hardcoded. It is better to place the configuration variable in environment or ignored file.

File: intellst-front/src/plugins/axios.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import Vue from "vue";
```
Comment (andreikojusko) [inline intellst-front/src/plugins/axios.js]: Probably, this file isn't needed,

**Review by andreikojusko — CHANGES_REQUESTED** [review]

**Comment by andreikojusko** [comment]
feel free to merge after resolving

File: intellst-front/public/index.html (hunk 1 above)
Comment (avolchivschi) [reply intellst-front/public/index.html]: Fixed

File: intellst-front/src/store/constants.js (hunk 2 above)
Comment (avolchivschi) [reply intellst-front/src/store/constants.js]: It's moved to the config.js file which is ignored

File: intellst-front/src/plugins/axios.js (hunk 3 above)
Comment (avolchivschi) [reply intellst-front/src/plugins/axios.js]: Deleted

**Pull request merged**